Branch-SDK/src/io/branch/referral/ServerRequestRegisterInstall.java

```diff
@@ -94,6 +95,7 @@ public void onRequestSucceeded(ServerResponse resp, Branch branch) {
             prefHelper_.setExternalIntentExtra(PrefHelper.NO_STRING_VALUE);
             prefHelper_.setAppLink(PrefHelper.NO_STRING_VALUE);
             prefHelper_.setPushIdentifier(PrefHelper.NO_STRING_VALUE);
+            prefHelper_.setIsAppLinkTriggeredInit(false);
```

Comment:
oops - I misread .. sorry. But you'll need this in the `onRequestSucceeded` for RegisterOpen too I believe, right?